### PR TITLE
Floating action button done

### PR DESCRIPTION
**Pull request by Amoodaa**
relates #18

### CODE REVIEW

File: client/src/components/Fab/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { Icon } from 'antd';
```
Comment (mossa-Sammer) [inline client/src/components/Fab/index.js]:
```suggestion
import { Icon } from 'antd';

```

File: client/src/components/Fab/index.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import './style.css';
```
Comment (mossa-Sammer) [inline client/src/components/Fab/index.js]:
```suggestion

import './style.css';
```

File: client/src/components/Fab/index.js (hunk 1 above)
Comment (mossa-Sammer) [inline client/src/components/Fab/index.js]:
```suggestion

```
remove unused components

File: client/src/components/Fab/style.css [hunk 3]
```diff
@@ -0,0 +1,11 @@
+.floating-action {
+  background-color: var(--main-color);
+  position: fixed;
+  border: 0px;
+  border-radius: 50%;
+  width: 30px;
+  height: 30px;
+}
+.floating-action:focus{
```
Comment (mossa-Sammer) [inline client/src/components/Fab/style.css]:
```suggestion

.floating-action:focus{
```

File: client/src/components/Fab/style.css [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+    outline: none;
+}
```
Comment (mossa-Sammer) [inline client/src/components/Fab/style.css]:
```suggestion
}

```

File: client/src/components/Fab/style.css [hunk 5]
```diff
@@ -0,0 +1,20 @@
+.circle-main-color-button {
+  background-color: var(--main-color);
+  border: 0px;
```
Comment (aminalakhsham) [inline client/src/components/Fab/style.css]: remove it

File: client/src/components/Fab/index.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import Button from '../Button';
```
Comment (mossa-Sammer) [inline client/src/components/Fab/index.js]:
```suggestion

import Button from '../Button';
```

**Pull request merged**